### PR TITLE
make it work

### LINKED ISSUE #162: TypeTheory does not compile with Coq 8.10
File "./TypeTheory/Initiality/Typing.v", line 63, characters 2-275:
Error:
Anomaly
"Ill-formed template inductive declaration: not polymorphic on any universe."
Please report at http://coq.inria.fr/bugs/.

---

COQC TypeTheory/ALV1/CwF_Defs_Equiv.v
does not compile either because use fails somewhere

### PR DESCRIPTION
**Pull request by DanGrayson**
We fix a few things so everything will work under the latest version of UniMath, and we modify the travis script so it works, too, and always uses the same version of UniMath (which should be periodically updated).

Resolves #162

### CODE REVIEW

**Comment by benediktahrens** [comment]
Thanks a lot for these improvements!

I am a little bit unsure about building against a fixed commit of UniMath vs always building against master. Ideally, TypeTheory would always compile with latest UniMath. The alternative would mean that differences accumulate, which make it harder to upgrade the longer one waits ("one" typically being me). An immediate obligation to adapt TypeTheory to UniMath could be helpful.

**Comment by DanGrayson** [comment]
As you wish, just let me know.

One argument against always building against master is that then contributers should build against master, too.

One argument in favor of periodic updates is that a contributer who submits a pull request will not be surprised.

File: .travis.yml [hunk 1]
```diff
@@ -13,12 +13,14 @@ install:
...
   - opam install lablgtk camlp5 num --yes --verbose
 before_script:
-  - WPATH=$PWD
-  - cd ..        
-  - git clone https://github.com/UniMath/UniMath
-  - cd UniMath && git show && make PACKAGES="Foundations MoreFoundations Combinatorics Algebra NumberSystems CategoryTheory Topology Ktheory PAdics" install
+  - pushd ..        
+  - git clone --no-checkout https://github.com/UniMath/UniMath
+  - cd UniMath
+  - git checkout 2dadfb61f5ef0d9805cf0eb6b80ef2beb26472d5
```
Comment (benediktahrens) [inline .travis.yml]:
```suggestion
  - git checkout master
```

Comment (benediktahrens) [reply .travis.yml]: I'd like to see how this works, building against master. If it does not work well, I am happy to revert to building against a fixed commit.

**Comment by peterlefanulumsdaine** [comment]
LGTM.  Like Benedikt, I support building against UniMath master, in order to keep TypeTheory in sync with upstream changes in UniMath.

**Comment by benediktahrens** [comment]
@DanGrayson : Thanks for preparing this!

**Pull request merged**